DEVELOPMENT.md

```diff
@@ -0,0 +1,48 @@
+## General requirements
+- Java 17 (should be specified in JAVA_HOME)
+- [macOs/Linux] Android SDK downloaded via `./jbdeps/android-sdk/downloadAndroidSdk`
+- [Windows] Android SDK downloaded from Android Studio and specified in ANDROID_SDK_ROOT
+- [Optional, macOs] For AppCode IDE, specify environment variable `export ANDROID_SDK_ROOT=[YOUR_PATH]/jbdeps/android-sdk/darwin` (in ~/.zshrc)
+
+## Developing in IDE
+1. Download Android Studio from [the official site](https://developer.android.com/studio/archive) (it is mandatory to use the version, written [here](https://github.com/JetBrains/androidx/blob/jb-main/gradle/libs.versions.toml#L11)). As an alternative you can use IDEA, which is compatible with [this AGP version](https://github.com/JetBrains/androidx/blob/jb-main/gradle/libs.versions.toml#L5), or you can disable Android plugin in IDEA plugins, to develop non-Android targets.
+2. [macOs/Linux] Download Android SDK via `./jbdeps/android-sdk/downloadAndroidSdk`
+3. [Windows] Download Android SDK via [Android Studio](https://developer.android.com/studio/intro/update#sdk-manager) and specify it in ANDROID_SDK_ROOT environment variable. Components, their versions and folder structure should be the same as in downloaded via script `./jbdeps/android-sdk/downloadAndroidSdk` SDK for other platforms.
+4. Specify Gradle JVM to use JDK 17 in InteliJ IDEA Preferences (`Build, Execution, Deployment -> Build Tools -> Gradle`)
+
+### Run tests
+
+Run tests for Desktop:
+```bash
+./gradlew :mpp:testDesktop
+```
+
+Run tests for Web:
+```bash
+./gradlew :mpp:testWeb
+```
+
+Run tests for UIKit:
+```bash
+./gradlew :mpp:testUIKit
+```
+
+### Run samples
+Run jvm desktop sample:
+```bash
+./gradlew :mpp:run
+```
+another jvm desktop samples:
+```bash
+./gradlew :mpp:run1 :mpp:run2 :mpp:run3 :mpp:runSwing :mpp:runWindowApi :mpp:runVsync :mpp:runLayout
+```
+
+Run wasm sample:
+```bash
+./gradlew :mpp:runMppJs
+```
+
+Run native macos sample:
+```bash
+./gradlew :mpp:runMppMacos
```

Comment:
JFYI: I have a weird error when running the examples via :mpp:runMppJs or :mpp:runMppMacos
```
org.gradle.api.internal.initialization.DefaultClassLoaderScope@139f8295 must be locked before it can be used to compute a classpath!
```

At the same time, when I run them as `:compose:mpp:demo:jsRun` or `:compose:mpp:demo:runDebugExecutableMacosX64`
___
Unless you have/had the same issue, I think there's no actions points for now. I think it's something weird on my machine.

Reply:
Just checked, I have the same issue, hm

Reply:
Really weird error, I tried to remove `buildscript` part, to convert :compose:mpp gradle build file to kotlin, and some other things, but it still doesn't work... Investigating further

Reply:
Anyway, we already simplify it with direct task run `:compose:mpp:demo:jsRun`

Reply:
You are right :D